<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure XML Filter</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox7">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="ExtractSemantic">
        <property name="toolTip">
         <string>Index the tags and the attributes.</string>
        </property>
        <property name="text">
         <string>Extract tags</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="UseDefaultNamespace">
          <property name="toolTip">
           <string>Specify if a default namespace must be used.</string>
          </property>
          <property name="whatsThis">
           <string>When the structural tokens are extracted, they are indexed with their namespace. If no namespace is defined, the URL of the file is supposed to be a local namespace. The user may specify a particular namespace to be used as default.</string>
          </property>
          <property name="text">
           <string>Default Namespace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DefaultNamespace"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Metadata</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="ExtractMetadata">
        <property name="toolTip">
         <string>Index the metatags.</string>
        </property>
        <property name="text">
         <string>Extract metadata?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="DetectMetadataTag">
        <property name="toolTip">
         <string>Should the metatags be detected, or only those explicitly specified are used.</string>
        </property>
        <property name="text">
         <string>Detect metadata tags?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="MetadataTags"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="AddMetadataTag">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="RemoveMetadataTag">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="MetadataDetection">
        <property name="title">
         <string>Metadata Detection Constraints</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="MetadataChildTags">
           <property name="toolTip">
            <string>Can a detected metadata tag have child tags?</string>
           </property>
           <property name="text">
            <string>Child Tags</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_2">
           <item>
            <widget class="QLabel" name="textLabel1_3">
             <property name="text">
              <string>Maximum Terms</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="MetadataMaxTerms"/>
           </item>
           <item>
            <spacer name="spacer12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_3">
           <item>
            <widget class="QLabel" name="textLabel2_3">
             <property name="text">
              <string>Maximum Depth</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="MetadataMaxDepth"/>
           </item>
           <item>
            <spacer name="spacer13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_4">
           <item>
            <widget class="QLabel" name="textLabel1_5">
             <property name="text">
              <string>Maximum Occurences</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="MetadataMaxOccurs"/>
           </item>
           <item>
            <spacer name="spacer15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Tags which content must not be extracted</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="ExcludeTags"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="AddExcludeTag">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="RemoveExcludeTag">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ImportTags">
     <property name="toolTip">
      <string>Import a XML file containing definitions of tags</string>
     </property>
     <property name="whatsThis">
      <string>Import a XML file which contain definition of tags. In practice, the file must be formed as:
&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf8&quot;?&gt;
&lt;tags&gt;
&lt;exclude&gt;http://mytags.org:support&lt;/exclude&gt;
&lt;metadata&gt;http://purl.org/dc/elements/1.1:title&lt;/metadata&gt;
&lt;metadata&gt;http://purl.org/dc/elements/1.1:author&lt;/metadata&gt;
&lt;/tags&gt;</string>
     </property>
     <property name="text">
      <string>Import Tag Definitions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>UseDefaultNamespace</sender>
   <signal>toggled(bool)</signal>
   <receiver>DefaultNamespace</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DetectMetadataTag</sender>
   <signal>toggled(bool)</signal>
   <receiver>MetadataDetection</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleStruct(bool)</slot>
  <slot>toggleFullIndex(bool)</slot>
 </slots>
</ui>
